tests/BUILD.txt

```diff
@@ -0,0 +1,7 @@
+Building aarch64-w64-mingw32 tests on Linux or WSL
+
+cmake -S . -B build
+cmake --build build
+
+it generates Arm64 Windows executable
```

Comment:
Why not a full sentence? (Starting with capital letter and ending with `.`.)

Reply:
Done